### PR TITLE
Rebranding styles

### PR DESCRIPTION
**Pull request by ticidesign**
screen shot 2018-10-25 at 12 01 38 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,52 +1,61 @@
...
+    <div>
+      <ul>
+        <li class="twitter">
+          <span>Twitter:</span>
+          <a href="https://twitter.com/MusesCodeJSSydney" target="_blank" rel="noopener noreferrer">@MusesCodeJSSydney</a>
+          <a href="https://twitter.com/MusesCodeJSMelb" target="_blank" rel="noopener noreferrer">@MusesCodeJSMelb</a>
+        </li>
+        <li class="facebook">
+          <span>Facebook:</span>
+          <a href="https://www.facebook.com/MusesCodeJSAustralia" target="_blank" rel="noopener noreferrer">MusesCodeJSAustralia</a>
```
Comment (ButenkoT) [inline index.html]: facebook is MusesCodeJS

File: css/main.css [hunk 2]
```diff
@@ -71,17 +76,17 @@ main a {
...
 }
 
 footer > div {
-    background-color: black;
+    background-color: #464646;
     color: white;
     display: flex;
     flex-direction: row;
-    font-family: Georgia, serif;
+    font-family: 'Source Sans Pro', sans-serif;
```
Comment (ButenkoT) [inline css/main.css]: our second font, and font for instructions should be 'Avenir'

**Comment by ButenkoT** [comment]
Also can you add screenshot to description :) Thanks!

File: css/main.css (hunk 2 above)
Comment (ticidesign) [reply css/main.css]: Avenir ins't a google font https://fonts.google.com/?query=Avenir

**Comment by ButenkoT** [comment]
I think we can do footer of one of our main colors? What do you think? Or it looks too grey

File: css/main.css (hunk 2 above)
Comment (ButenkoT) [reply css/main.css]: You can put Avenir as 1st choice, then keep Source Sans Pro. It works even without adding google font link

**Pull request merged**